client/src/Components/Common/Comment/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Row, Col, Typography } from 'antd';
+import PropTypes from 'prop-types';
+
+import Rating from '../Rating';
+import MainButton from '../MainButton';
+import UserImage from '../UserImage';
+import './style.css';
+
+const { Title, Text } = Typography;
+
+const Comment = ({
+  isAdmin,
+  userName,
+  maxWidth,
+  commentText,
+  date,
+  rateValue,
+  onAdminClick,
+  ...otherProps
+}) => (
+  <div className="Comment" style={{ maxWidth }} {...otherProps}>
+    <Row justify="start">
+      <Col id="avatar" span={5}>
+        <Row>
+          <UserImage size={64} userName={userName} />
+        </Row>
+        <Title level={5}>{userName}</Title>
+      </Col>
+      <Col span={19}>
+        <Row className="top-container">
+          <Rating rateValue={rateValue} />
+          <div id="comment-date">{date}</div>
```

Comment:
change the div to an appropriate heading `h3` for example and style it to be displayed `inline`